extensions/ql-vscode/src/upgrades.ts

```diff
@@ -14,79 +18,101 @@ import { upgradesTmpDir } from './run-queries';
 const MAX_UPGRADE_MESSAGE_LINES = 10;
 
 /**
- * Checks whether the given database can be upgraded to the given target DB scheme,
- * and whether the user wants to proceed with the upgrade.
- * Reports errors to both the user and the console.
- * @returns the `UpgradeParams` needed to start the upgrade, if the upgrade is possible and was confirmed by the user, or `undefined` otherwise.
+ * Check that we support non-destructive upgrades.
+ * 
+ * This requires 3 features. The ability to compile an upgrade sequence; The ability to 
+ * run a non-destructive upgrades as a query; the ability to specify a target when 
```

Comment:
Three features are mentioned here, but this function only checks for one of them. Can you clarify in this comment what is actually being checked here.